src/routes/safe/container/index.tsx

```diff
@@ -24,6 +24,14 @@ const Balances = React.lazy(() => import('../components/Balances'))
 const TxsTable = React.lazy(() => import('src/routes/safe/components/Transactions/TxsTable'))
 const AddressBookTable = React.lazy(() => import('src/routes/safe/components/AddressBook'))
 
+const featuresEqualityFn = (left, right) => {
```

Comment:
hmm... why does it not complain about the types? :P 

Also, I think It could be a generic util function, not only featuresEquality

Reply:
Actually, even a better solution would be removing enabledFeatures from the redux store... It can be computed only with a version number, doesn't have to be stored.

Reply:
agreed. But let's do it in a different PR once xDai is merged.

Reply:
Why? It's not a huge change. The amount of tech debt tickets we have is pretty high (and there's a ton of discussed, but not created ones), so I'd think before adding a new one

Reply:
```
  const featuresEnabled = useMemo(() => enabledFeatures(safe.version), [safe.version])
```

literally, that's it, except for removing the calculation in `fetchSafe.ts, updating types and replacing references. It's around 10 mins of work.

Reply:
@mikheevm [`56ad1bc` (#1477)](https://github.com/gnosis/safe-react/pull/1477/commits/56ad1bc92f04e40c6e11c98b87f63427dd590f57)